src/app/services/form-storage.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { STORAGE_KEY } from '@interface/form-data';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FormStorageService {
+  setItem(key: STORAGE_KEY, value: any): void {
```

Comment:
Avoid the use of the any type. Always ensure that your application is typesafe

Reply:
Ok